common/src/main/java/org/terracotta/angela/common/topology/Version.java

```diff
@@ -35,7 +35,24 @@ public static Version version(String version) {
     return new Version(version);
   }
 
+  public static Version rawVersion(String version) {
```

Comment:
The distribution controller angela creates are based on these versions.

We had explicitly added support for `terracotta-5.7.zip` kit and agreed that we'd use this name for the new kit. I see this changes as both unnecessary and dangerous.

Reply:
this requires renaming the vanilla kit to terracotta-5.7... Is this what we really want ? The name collde with the core base kit

Reply:
Also, I disagree that each time we have a version, kit or project we have to update angela.

I would like angela to be able to run on any kit I provide. This is not normal to have hardcoded names and versions in the code IMO.

Reply:
The support for 5.7 kit is added in angela. The name of the kit doesn't matter.

>> Also, I disagree that each time we have a version, kit or project we have to update angela.
We don't need to do it every time. Whatever changes were needed in Angela for versioning are done, and won't require further changes for each released version.

>> This is not normal to have hardcoded names and versions in the code IMO.
I'm with you. It's the way angela works. Feel free to rewrite it ;-)

>> We decided to use angela if we were able to use it as a drop in replacement of what we had before
It'd work correctly if properties are correctly set. Let me know if the properties don't work out.

Reply:
angela oss or ee ?